### ISSUE TITLE
Custom templates?

### ISSUE THREAD

**Issue by mAAdhaTTah** [kitchenplan/chef-dotfiles#19]
You've got all custom bash scripts for bash-it; wondering if there's some way to replaces them with our own from our kitchenplan-config.

**Comment by roderik** [comment]
Wel, nothing included at least. I'm also thinking of moving away from bash_it but don't have the time to go for something more generic.

**Comment by mAAdhaTTah** [comment]
Interesting, why are you considering moving away from it? I'm just trying to settle on a solution to integrating my .dotfiles repo into a Kitchenplan provision that doesn't require the extra step and downloading and bootstrapping them. The only thing KP suffers from is it's still wedded somewhat to your specific implementation, so if there's something I could do to help here, let me know. Unfortunately, I'm not really a rubyist, so I don't know how much help I can be.

**Comment by roderik** [comment]
Mostly, the quest for something shiny and new :) Everything works like i want it to at the moment so it never goes further than the idea

**Comment by dsbaars** [comment]
I want to suggest to comment everything out in the template:
(https://github.com/kitchenplan/chef-dotfiles/blob/master/templates/default/ssh_config.erb).

I was quite suprised those lines got injected into /etc/ssh_config.. Very happy kitchenplan is verbose so I noticed it right away :)

**Comment by roderik** [comment]
Yeah, that's a bad one :fearful: There is some confusion with this one https://github.com/kitchenplan/chef-applications/blob/master/recipes/ssh_config.rb where everything is configurable, but it's in the wrong cookbook to be "logical".
